glusterd2/pmap/rpc_prog.go

```diff
@@ -138,7 +138,9 @@ func (p *GfPortmap) SignIn(args *SignInReq, reply *SignInRsp) error {
 		"port":    args.Port,
 	}).Debug("brick signed in")
 
-	registry.Update(args.Port, args.Brick, conn)
+	// TODO: Add Pid field to SignInRsp and pass it here when
```

Comment:
@prashanthpai Question : We also need to add Pid field to SignInReq as well, right ?

Reply:
Oops, I meant `SignInReq` not `SignInRsp`

Reply:
That change can only be done after https://review.gluster.org/#/c/glusterfs/+/21503/ is merged and it ends up in the nightly RPMs

Reply:
Fixed the comment now